### ISSUE TITLE
Typo in ngb-cli help

### ISSUE THREAD

**Issue by jlncrnt** [epam/NGB#1]
In the following section in ngb-cli man page:
```
FILE commands:
rf	ref_file	: registers a feature file for a specified reference	{rf grch38 \path\to\file.bam?\path\to\file.bam.bai -n my_vcf}
df	del_file	: deletes a feature file one	{df my_vcf}
if	index_file	: creates a feature index for a file. 	 {if genes.gtf}
```
shouldn't `ref_file` be `reg_file` ?

**Comment by sidoruka** [comment]
@jlncrnt thanks for pointing this out. 
Right, `reg_file` is a correct command.
This typo will be fixed in the nearest update early next week.

**Comment by sidoruka** [comment]
Typo is fixed in the c3522e5 : [Application.java#L68](https://github.com/epam/NGB/blob/master/server/ngb-cli/src/main/java/com/epam/ngb/cli/app/Application.java#L68)
